pkg/controller/releasemanager/garbagecollector.go

```diff
@@ -47,7 +47,11 @@ func markDeletable(ctx context.Context, log logr.Logger, cli client.Client, inca
 	if revision.Labels[label] != "true" {
 		log.Info("Marking revision for deletion", "revision", revision, "randomTarget", revision.Spec.Targets[0])
 		revision.Labels[label] = "true"
-		return cli.Update(ctx, revision)
+		for err := cli.Update(ctx, revision); err != nil {
```

Comment:
```suggestion
		if err := cli.Update(ctx, revision); err != nil {
```
I… think this is equivalent?

Reply:
lol